### PR TITLE
feature: upgrade React to 18

### PR DESCRIPTION
**Pull request by donniean**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -68,12 +68,12 @@
     "@commitlint/cli": "^16.2.4",
     "@commitlint/config-conventional": "^16.2.4",
     "@emotion/babel-plugin": "^11.9.2",
-    "@pmmmwh/react-refresh-webpack-plugin": "^0.5.5",
+    "@pmmmwh/react-refresh-webpack-plugin": "^0.5.6",
     "@stylelint/postcss-css-in-js": "^0.38.0",
     "@svgr/webpack": "^6.2.1",
     "@types/axios": "^0.14.0",
```
Comment (donniean) [inline package.json:74]: test

Comment (donniean) [reply package.json:74]: test apply

**Review by jackzerone — CHANGES_REQUESTED** [review]
approve

**Review by jackzerone — APPROVED** [review]

**Pull request merged**